src/main/resources/vm/jdl/dirac-jdl.vm

```diff
@@ -2,13 +2,17 @@
 ##
 ## Variables
 ## $jobName, $scriptPath, $scriptName, $cpuTime,
-## $priority, $site $bannedSite, $tags
+## $priority, $site, $bannedSite, $tags
 ##
 JobName         = "$jobName";
 Executable      = "$scriptName";
 StdOutput       = "std.out";
 StdError        = "std.err";
-InputSandbox    = {"$scriptPath/$scriptName"};
+InputSandbox    = {"$scriptPath/$scriptName"
+#if($isMoteurliteEnabled)
```

Comment:
i find it prettier/clearer to put both `"$scriptPath/$scriptName"` in both case.

Reply:
done